DevEdu.API/DevEdu.API/Configuration/ExceptionMiddleware.cs

```diff
@@ -32,23 +32,23 @@ public async Task Invoke(HttpContext context)
             }
             catch (AuthorizationException ex)
             {
-                await HandlerExceptionMessageAsync(context, ex, AuthorizationCode, MessageAuthorization);
+                await HandlerExceptionMessageAsync(context, ex, AuthorizationCode, MessageAuthorization, HttpStatusCode.Forbidden);
             }
-            catch (ValidationException ex) //422
+            catch (ValidationException ex)
             {
                 await HandleValidationExceptionMessageAsync(context, ex, ValidationCode, MessageValidation);
             }
-            catch (EntityNotFoundException ex) //404
+            catch (EntityNotFoundException ex)
             {
-                await HandlerExceptionMessageAsync(context, ex, EntityCode, MessageEntity);
+                await HandlerExceptionMessageAsync(context, ex, EntityCode, MessageEntity, HttpStatusCode.NotFound);
             }
             catch (Exception ex)
             {
                 await HandleExceptionMessageAsync(context, ex);
             }
         }
 
-        private static Task HandlerExceptionMessageAsync(HttpContext context, Exception exception, int code, string message)
+        private static Task HandlerExceptionMessageAsync(HttpContext context, Exception exception, int code, string message, HttpStatusCode httpStatusCode)
```

Comment:
hmmmm... too many params here now...
you can pass only statusCode and get the needed code and message by statusCode (you can use a ternary operator)

and please rename private consts according to naming conventions